### ISSUE TITLE
Overriding config file

### ISSUE THREAD

**Issue by erik-stephens** [boxen/puppet-dnsmasq#15]
Issue #5 references a way to override the config via dnsmasq::config class but that class [was removed](https://github.com/boxen/puppet-dnsmasq/commit/50c2ec7f4681654ae6a8ad71c0bf42e91a739360).  There is [reference to a $configfile param](https://github.com/boxen/puppet-dnsmasq/blob/master/manifests/init.pp#L11) but it is not used.  I'm going to fix in my personal fork.  If would like me to submit a PR, help me understand how `$configfile` was intended to be used: a source, a template, or something else entirely?

**Comment by msturm** [comment]
Hi,
What is the plan to fix the overriding of the dnsmasq.conf file? Because now I have to do a bit of a nasty hack to add additional entries to this file, but a nice way to override it would be cool... The repository of erik-stephens doesn't work, because it doesn't start dnsmasq with the correct config file (that is probably the reason why that $configfile variable is still there...)

**Comment by erik-stephens** [comment]
Yeah, not sure why I commented out $configfile in my fork, maybe to indicate that it's not used.  I think I searched for `$configfile` but after looking again, I see it referenced in an erb template as `@configfile` - lesson learned when grep'ing puppet modules.  Not tested but I did [undo that comment](https://github.com/erik-stephens/puppet-dnsmasq/releases/tag/2.1.1) so the daemon should start now.

Longer term, maybe have the config file driven by a hash variable?  I'm not sure worth the effort.  Putting the onus of managing the config file on the boxen user and not this module might be best.
